packages/theme/src/cli/commands/theme/dev.ts

```diff
@@ -131,31 +129,20 @@ export default class Dev extends ThemeCommand {
       return
     }
 
-    let controller = new AbortController()
-
-    setInterval(() => {
-      outputDebug('Refreshing theme session token and restarting theme server...')
-      controller.abort()
-      controller = new AbortController()
-      // eslint-disable-next-line @typescript-eslint/no-floating-promises
-      this.execute(adminSession, flags.password, command, controller, true)
-    }, this.ThemeRefreshTimeoutInMs)
-
     renderLinks(store, flags.theme!, flags.host, flags.port)
 
-    // eslint-disable-next-line @typescript-eslint/no-floating-promises
-    this.execute(adminSession, flags.password, command, controller, false)
+    // eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
I've been reading about this, but I'm not sure if there's a better way to do this (although it works)...